app/scripts/polling_location_finder.js

```diff
@@ -78,8 +86,101 @@ define(['jquery', 'geojson', 'json!vendor/ELECTIONS_WardsPrecincts.geojson', 'js
         return encodeURI(url + destination);
     }
 
-    return function(latLng, successCallback, errorCallback) {
+    function getHours(place) {
+
+        var hours = [];
+        moment.tz.add('America/New_York|EST EDT|50 40|0101|1Lz50 1zb0 Op0');
+        
+        jQuery.each(place, function(index, timeInterval) {
+            var range = moment.range(timeInterval);
+            var dates = range.toDate();
+            dates = dates.map(function(d) { 
+                return moment(d).tz('America/New_York').format('MMM Do h:mma z')
+            });
+
+            hours.push(dates);
+
+         });
+
+        return hours;
+    }
+
+    function displayEarlyPollingLocations() {
+
+       
+
+        for (var i = 0; i < earlyPollingLocations.length; i++) {
+   
+            var pos = new google.maps.LatLng(earlyPollingLocations[i].position.lat(),
+                                             earlyPollingLocations[i].position.lng());
+
+            var addr = earlyPollingLocations[i].geojsonProperties.Full_Addr;
+
+            var hours = earlyPollingLocations[i].geojsonProperties.hours;
+
+            var listed_hours = getHours(hours);
+            createEarlyVotingMarker(pos, i, addr, listed_hours);
+        }
+    }
+
+    function stripHourInfo(hourString) {
+        var spaceCount = 2;
+        var i = 0;
+        while(spaceCount > 0) {
+            if (hourString[i] == " ") {
+                spaceCount--;
+            }
+            i++;
+        }
+        return (hourString.substring(i, hourString.length-4));
+    }
+
+    function generateHoursHTML(listed_hours) {
+        var hoursHTML = "";
+        for (var i = 0; i < listed_hours.length; i++) {
+         
+            var hourString = listed_hours[i][0];
+            hoursHTML += "<li>" + hourString.substring(0, hourString.length-4) 
+                      + " to " + stripHourInfo(listed_hours[i][1]) + "</li>";
+        }
+        return hoursHTML;
+
+    }
+
+    function createEarlyVotingMarker(pos, index, addr, listed_hours) {
+
+        var earlyVotingMarker = new google.maps.Marker({
+            position: pos,
+            map: map
+        });
+
+        var earlyVotingInfoWindow = new google.maps.InfoWindow();
+
+        var earlyVotingInfo = document.createElement("ul");
+        earlyVotingInfo.id = "earlyVotingInfoDiv";
+
+        var hoursHTML = generateHoursHTML(listed_hours);
+        earlyVotingInfo = "<div class='earlyPollingWindow'><b><h5 style='color:#000'>Early Polling Location: " 
```

Comment:
yup! I was originally going to style it differently and forgot to delete that line